lib/bolt/analytics.rb

```diff
@@ -139,6 +141,18 @@ def report_bundled_content(mode, name)
         end
       end
 
+      def plan_counts(plans_path)
+        pp_count, yaml_count = if File.exist?(plans_path)
+                                 %w[pp yaml].map do |extension|
+                                   Find.find(plans_path.to_s).grep(/.*\.#{extension}/).length
```

Comment:
The `Find` class is only available because we require the module in the applicator and orch transport. I think we should require it in this file as well just in case that changes in the future.